ocaml/parsing/builtin_attributes.ml

```diff
@@ -508,9 +508,11 @@ let zero_alloc_attribute (attr : Parsetree.attribute)  =
   parse_attribute_with_ident_payload attr
     ~name:"zero_alloc" ~f:(function
       | "check" -> Clflags.zero_alloc_check := true
+      | "assert-all" | "assert" | "assert_all" | "all" ->
+        Clflags.zero_alloc_check_assert_all := true
```

Comment:
Now that we have different possible
values, I wonder whether something
should be done if inconsistent / unused
attributes are present?

Reply:
There are some checks already in `Translattribute`. I'll add a test.